iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/ReplaceThreeWindingsTransformersBy3TwoWindingsTransformersTest.java

```diff
@@ -0,0 +1,318 @@
+        TwoWindingsTransformer t2w3 = network.getTwoWindingsTransformer("3WT-Leg3");
+
+        assertTrue(compareAliases(t2w1, "1"));
+        assertTrue(compareAliases(t2w2, "2"));
+        assertTrue(compareAliases(t2w3, "3"));
+    }
+
+    private void addLegAliases(String leg) {
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("transformerEnd-Leg" + leg, "CGMES.TransformerEnd" + leg);
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("terminal-Leg" + leg, "CGMES.Terminal" + leg);
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("ratioTapChanger-Leg" + leg, "CGMES.RatioTapChanger" + leg);
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("phaseTapChanger-Leg" + leg, "CGMES.PhaseTapChanger" + leg);
+    }
+
+    private boolean compareAliases(TwoWindingsTransformer t2w, String leg) {
+        return t2w.getAliasFromType("CGMES.TransformerEnd1").orElseThrow().equals("transformerEnd-Leg" + leg)
+                && t2w.getAliasFromType("CGMES.Terminal1").orElseThrow().equals("terminal-Leg" + leg)
+                && t2w.getAliasFromType("CGMES.RatioTapChanger1").orElseThrow().equals("ratioTapChanger-Leg" + leg)
+                && t2w.getAliasFromType("CGMES.PhaseTapChanger1").orElseThrow().equals("phaseTapChanger-Leg" + leg);
+    }
+
+    @Test
+    void testReportNode() throws IOException {
+        network.getThreeWindingsTransformer(t3w.getId()).setProperty("t3w property1", "t3w-value1");
+        network.getThreeWindingsTransformer(t3w.getId()).setProperty("t3w property2", "t3w-value2");
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("t3w-alias1");
+        network.getThreeWindingsTransformer(t3w.getId()).addAlias("t3w-alias2");
+        ReportNode reportNode = ReportNode.newRootReportNode().withMessageTemplate("test", "test reportNode").build();
+        ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers replace = new ReplaceThreeWindingsTransformersBy3TwoWindingsTransformers();
+        replace.apply(network, reportNode);
+
+        StringWriter sw1 = new StringWriter();
+        reportNode.print(sw1);
+        assertEquals("""
+                + test reportNode
+                   + Replaced ThreeWindingsTransformer by 3 TwoWindingsTransformers
+                      ThreeWindingsTransformer 3WT removed
+                      Property [t3w property2,t3w property1] of threeWindingsTransformer 3WT will be lost
+                      Alias [t3w-alias1,t3w-alias2] of threeWindingsTransformer 3WT will be lost
+                      VoltageLevel 3WT-Star-VL created
+                      TwoWindingsTransformer 3WT-Leg1 created
+                      TwoWindingsTransformer 3WT-Leg2 created
+                      TwoWindingsTransformer 3WT-Leg3 created
+                """, TestUtil.normalizeLineSeparator(sw1.toString()));
+    }
+
+    @Test
+    void testReportNodeExtensions() throws IOException {
```

Comment:
Consider for more consistency to name the method ```reportNodeExtensionsTest``` instead

Reply:
Done